examples/header.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+"""
+This example shows the functionality of the PageHeader object.
+
+It creates a sample page with the different types of headers.
+
+..  :copyright: (c) 2016 by Vladimir Gorovikov
+    :license: MIT, see License for more details.
+"""
+
+# begin-doc-include
+from pylatex import Document, PageStyle, Head, MiniPage
+from pylatex.utils import line_break, display_page_number, header1, header2, \
+    bold
+
+
+def generate_header():
+    geometry_options = {"margin": "0.5in"}
+    doc = Document(geometry_options=geometry_options)
+    # Add document header
+    header = PageStyle("header")
+    # Create left header
+    with header.create(Head("L")) as left_header:
+        left_header.append("Page date: ")
```

Comment:
You can use `header.append(` instead of `left_header.append(`, that way you don't need to use the as keyword. You only need it when you use other methods than `append` or `create`.